### DISCUSSION TITLE
Altyapı sorunu yaşayan yerlerin haritalanması

### DISCUSSION THREAD

**Discussion by sercangoger** [acikkaynak/deprem-yardim-projesi#60] [Ideas]
konum bilgisi istenilerek alt yapı sorunu olan yerleri belirlemek ve harita göstermek üzerine bir proje geliştirmeliyiz.

Elektrik 
Su 
Dogalgaz 
Internet

vb durumları harita da pinlemek
bazı bölgeler de internet çekerken bazı yerlerde çekmeye bilir.
veya bir yerde ısınma sorun olmazken bir yerde olabilir. 
elektrik olan yerler de insanlar daha hızlı iletişim kurması olası olacaktır.
Sonuclar bu şekilde arttırılabilir.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebiniz Legal ekibin kontrolü sonrasında reddedildi.
Notları: "Legal ret, Çevre şehircilik bakanlığı konusu"
